druid-shell/src/backend/windows/window.rs

```diff
@@ -1882,14 +1886,30 @@ impl WindowHandle {
         self.defer(DeferredOp::ShowTitlebar(show_titlebar));
     }
 
-    // Sets the position of the window in virtual screen coordinates
     pub fn set_position(&self, position: Point) {
         self.defer(DeferredOp::SetWindowState(window::WindowState::Restored));
-        self.defer(DeferredOp::SetPosition(position));
-    }
-
-    pub fn set_level(&self, _level: WindowLevel) {
-        warn!("Window level unimplemented for Windows!");
+        if let Some(w) = self.state.upgrade() {
+            if let Some(window_level) = w.window_level.borrow().as_ref() {
+                match window_level {
+                    WindowLevel::Tooltip(parent_window_handle)
+                    | WindowLevel::DropDown(parent_window_handle)
+                    | WindowLevel::Modal(parent_window_handle) => {
+                        // Has owned window. Convert point from window coords to screen coords.
+                        let screen_position =
+                            parent_window_handle.get_position() + position.to_vec2();
+                        self.defer(DeferredOp::SetPosition(screen_position));
+                    }
+                    _ => {
+                        // Dead code!! Not sure if app window will ever be owned but we can be defensive.
```

Comment:
Ah ok, I see what the `Option<WindowLevel>` is doing now. I think I prefer getting rid of the option still, because then the types will get rid of this dead code.

Reply:
Default as `AppWindow` made this easier. Fixed this and your suggestion below. Thanks.